flaskapp.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask, json
+from flask_restful import reqparse, abort, Api, Resource
+from app import PublicNeeds
```

Comment:
app should be flask app, and the PublicNeeds should be included in module file called differently than app.py